anesthetic/plot.py

```diff
@@ -264,11 +272,15 @@ def fastkde_plot_1d(ax, data, *args, **kwargs):
     if data.max()-data.min() <= 0:
         return
 
+    filled1d = kwargs.pop('filled1d', False)
+    levels = kwargs.pop('levels', [0.68, 0.95])
+
     xmin = kwargs.pop('xmin', None)
     xmax = kwargs.pop('xmax', None)
     cmap = kwargs.pop('cmap', None)
     color = kwargs.pop('color', (next(ax._get_lines.prop_cycler)['color']
                                  if cmap is None else cmap(0.68)))
+    facecolor = kwargs.pop('facecolor', color)
```

Comment:
Could facecolor act as the same as filled1d, defaulting to None, but if it is equal to True, then taking color as its default? I don't think facecolor is used otherwise. This would mean we wouldn't need 'filledkde' as an option to plot_1d.

Reply:
@williamjameshandley ``facecolor`` is used if ``cmap`` is ``None`` and ``filled1d`` is ``True``  to determine the color map. We could remove ``filled1d`` and replace,

```python
if filled1d is True:
        c = iso_probability_contours(p[i], contours=levels)
        if cmap is None:
            cmap = basic_cmap(facecolor)
        for j in range(len(c)-1):
            ax.fill_between(x[i], p[i], where=p[i] >= c[j],
                            color=cmap(c[j]))
```
with
```python
if any([facecolor, cmap]) is not None:
        c = iso_probability_contours(p[i], contours=levels)
        if cmap is None:
            cmap = basic_cmap(facecolor)
        for j in range(len(c)-1):
            ax.fill_between(x[i], p[i], where=p[i] >= c[j],
                            color=cmap(c[j]))
```
and set ``facecolor`` to None as a default. The user could then pass a color map or a facecolor from which to build a cmap to the function to trigger a filled plot?

Reply:
I would probably implement this by defaulting it to False in the kwargs.pop, and then using an `if facecolor` check in place of `if filled1d`, and then within that checking `if facecolor is True: facecolor=color`. I think that covers all bases.

Reply:
@williamjameshandley No problem that sounds good to me. See latest commit.